PhysicsTools/ONNXRuntime/interface/ONNXRuntime.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * ONNXRuntime.h
+ *
+ * A convenience wrapper of the ONNXRuntime C++ API.
+ * Based on https://github.com/microsoft/onnxruntime/blob/master/csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests.Capi/CXX_Api_Sample.cpp.
+ *
+ *  Created on: Jun 28, 2019
+ *      Author: hqu
+ */
+
+#ifndef PHYSICSTOOLS_ONNXRUNTIME_INTERFACE_ONNXRUNTIME_H_
+#define PHYSICSTOOLS_ONNXRUNTIME_INTERFACE_ONNXRUNTIME_H_
+
+#include <vector>
+#include <map>
+#include <string>
+#include <memory>
+
+#include "onnxruntime/core/session/onnxruntime_cxx_api.h"
+
+namespace cms {
+
+  namespace Ort {
```

Comment:
minor, but perhaps more elegant and requiring less indentation is to use ```namespace cms::Ort```

Reply:
Fixed in https://github.com/cms-sw/cmssw/pull/28112/commits/c02969dac6eda5d1ab6991c79a3156984faa7fd3.